### ISSUE TITLE
DOC: Error in the random generating syntax in the verision 1.26

### ISSUE THREAD

**Issue by LittleDogXi** [numpy/numpy#26485]
### Issue with current documentation:

I have found a little error in the NumPy documentation, version 1.26, NumPy for MATLAB users, Fundamental and usage, NumPy user guide. It locates as the picture below. [Website Link](https://numpy.org/doc/1.26/user/numpy-for-matlab-users.html)
![bug](https://github.com/numpy/numpy/assets/96516639/b8730348-84ad-46b6-92cd-3584ed0952bd)
The code "**rng.random(3, 4)**" is wrong, and should be corrected into "**rng.random((3, 4))**". A bracket is missed. I have just found it at about UTC 09:30, 5/20/2024.

**Comment by luxedo** [comment]
I just ran and confirmed the behavior.
```shell
Python 3.11.8 (main, Feb 12 2024, 14:50:05) [GCC 13.2.1 20230801] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from numpy.random import default_rng
>>> rng = default_rng(42)
>>> rng.random(3, 4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "_generator.pyx", line 289, in numpy.random._generator.Generator.random
TypeError: Cannot interpret '4' as a data type
>>> rng.random((3, 4))
array([[0.77395605, 0.43887844, 0.85859792, 0.69736803],
       [0.09417735, 0.97562235, 0.7611397 , 0.78606431],
       [0.12811363, 0.45038594, 0.37079802, 0.92676499]])
>>> 
```

The [documentation](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.random.html) is very clear as well. It only receives a single argument for the shape (size) of the output.
```python
random(size=None, dtype=np.float64, out=None)
```

**Comment by rossbar** [comment]
Fixed by #26942
